requirements/base.txt

```diff
@@ -2,3 +2,5 @@ aiohttp>=3.5.4
 aiodns>=2.0.0
 attrs==19.1.0
 maxminddb>=1.4.1
+PyOpenSSL
+git+https://github.com/benoitc/http-parser
```

Comment:
Is it good practice to add random commit as dependency? Maybe it would be better to pin latest commit in requirements.

Reply:
Not good practice, no. Unfortunately `http-parser` on PyPi is not working, and therefore I had no choice. No excuse for `PyOpenSSL`.

This is a temporary fix, however, and I'm currently on the works on re-writting the entire project with the man-in-the-middle structure.